includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -377,6 +367,12 @@ private function isMessageWallEnabled() {
 		return WikiFactory::getVarValueByName( 'wgEnableWallExt', $this->productInstanceId );
 	}
 
+	private function isWikiaOrgCommunity() {
+		return $this->product !== self::PRODUCT_WIKIS ?
```

Comment:
what about simplifying this condition?

`$this->product !== self::PRODUCT_WIKIS && WikiFactory::getVarValueByName( 'wgIsInWikiaOrgProgram', $this->productInstanceId )`